internal/shim-sgx/src/heap.rs

```diff
@@ -53,52 +67,26 @@ where
     /// This function is unsafe because it expects the block to be mapped RWX.
     const unsafe fn new(blk: &'a mut Block<N>) -> Self {
         let brk = None;
-        let bitmap = [0; N * 64];
         Self {
             blk,
-            map: bitmap,
             brk,
+            ledger: Ledger::new(Region::new(Address::new(0), Address::new(N * Page::SIZE))),
```

Comment:
I think we want absolute addressing here. All the requests (i.e. `brk()`, `mmap()`, etc) are in absolute addressing. Let's avoid unnecessary conversions.

Reply:
At this time there is no address.